viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx

```diff
@@ -112,66 +116,72 @@ export class TraceExplorerWidget extends ReactWidget {
-                    <List
-                        height={300}
-                        width={300}
-                        rowCount={outputsRowCount}
-                        rowHeight={50}
-                        rowRenderer={this.outputsRowRenderer} />
+            return <div className='trace-explorer-container'>
+                <ReactModal isOpen={this.showShareDialog} onRequestClose={this.handleShareModalClose}
+                    ariaHideApp={false} className='sharing-modal' overlayClassName='sharing-overlay'>
+                    {this.renderSharingModal()}
+                </ReactModal>
+                <div className='trace-explorer-opened'>
+                    <div className='trace-explorer-panel-title' onClick={this.updateOpenedExperiments}>
+                        {this.OPENED_TRACE_TITLE}
+                    </div>
+                    <div className='trace-explorer-panel-content'>
+                        <List
+                            height={300}
+                            width={300}
+                            rowCount={this.openedExperiments.length}
+                            rowHeight={50}
+                            rowRenderer={this.experimentRowRenderer} />
+                    </div>
                 </div>
-            </div>
-            <div className='trace-explorer-tooltip'>
-                <div className='trace-explorer-panel-title'>
-                    {'Time Graph Tooltip'}
+                <div className='trace-explorer-analysis'>
+                    <div className='trace-explorer-panel-title'>
+                        {this.ANALYSIS_TITLE}
+                    </div>
+                    <div className='trace-explorer-panel-content'>
+                        <List
+                            height={300}
+                            width={300}
+                            rowCount={outputsRowCount}
+                            rowHeight={50}
+                            rowRenderer={this.outputsRowRenderer} />
+                    </div>
                 </div>
-                <div className='trace-explorer-panel-content'>
-                    {this.renderTooltip()}
+                <div className='trace-explorer-tooltip'>
+                    <div className='trace-explorer-panel-title'>
+                        {'Time Graph Tooltip'}
+                    </div>
+                    <div className='trace-explorer-panel-content'>
+                        {this.renderTooltip()}
+                    </div>
                 </div>
+            </div>;
+        }
+
+        return <div className='theia-navigator-container' tabIndex={0}>
+            <div className='center'>{'You have not yet opened a Trace.'}</div>
+            <div className='open-workspace-button-container'>
+                <button className='theia-button open-workspace-button' title='Select a Trace to open'
+                    onClick={this.handleOpenTrace}>{'Open Trace(s)'}</button>
```

Comment:
> Apart from a couple details the visuals look good on my end! I went for maximum visual pickiness and I have no idea how difficult it is to implement these while keeping responsiveness, prioritizing elegant code, etc. So let me know. Especially for MVP I'm willing to loosen criteria to avoid code hacks.
> 
> Disclaimer for other reviewers: I just reviewed the visual style. I didn't run the code or check if it breaks anything.
> 
> Thanks for working on this! It's my first time reading code from this project and I understood your changes and what was going on so that says something positive about your code clarity 🍰

Thanks for reviewing the changes. I have updated few UI changes as per your suggestions.